src/OrchardCore/OrchardCore.Infrastructure/Documents/DocumentManager.cs

```diff
@@ -261,7 +261,7 @@ private async Task<TDocument> GetInternalAsync(bool failover = false)
                         return document;
                     }
 
-                    if (_isDistributed && (_options?.SlidingExpiration.HasValue ?? false))
+                    if (_isDistributed && _options.SlidingExpiration.HasValue)
```

Comment:
If `_options` is `null`, `_options?.SlidingExpiration.HasValue` will also be `null`.

Reply:
It wouldn't be null, check line 255

Reply:
I see. What about `SlidingExpiration` though?

Reply:
While we checking using `HasValue` nothing to worry, the old `??` operator was for `_option`